src/dftbp/dftb/hybridxc.F90

```diff
@@ -525,12 +525,12 @@ subroutine THybridXcFunc_init(this, nAtom0, species0, hubbu, screeningThreshold,
     if (present(gammaCutoff)) then
       this%gammaCutoff = gammaCutoff
 
-      ! Start beginning of the damping region and 95% of the gamma cutoff.
-      this%gammaDamping = 0.95_dp * this%gammaCutoff
+      ! Start of the damping region of the truncated Coulomb kernel
+      this%gammaDamping = this%gammaCutoff - distFudge
 
       if (this%gammaDamping <= 0.0_dp) then
-        @:RAISE_ERROR(errStatus, -1, "Beginning of damped region of electrostatics must be&
-            & positive.")
+        @:RAISE_ERROR(errStatus, -1, "The beginning of the damping region of the truncated Coulomb&
+            & kernel must be at positive radii.")
```

Comment:
```suggestion
            & kernel must be at a positive radius.")
```
Or is this not for a single value?

Reply:
In my mind I was evaluating the TCI kernel for several different radii and went with the plural, but your suggestion might be more intuitive.

Applied.